### PR TITLE
Add generate_announcements command

### PR DESCRIPTION
**Pull request by deciphr**
Generates a boilerplate announcement from Discord events.

### CODE REVIEW

**Comment by deciphr** [comment]
make sure you replace the emoji with the emoji in the server

\:cowboysunglasses:

File: cyberham/__main__.py [hunk 1]
```diff
@@ -530,5 +533,68 @@ async def delete_all_events(interaction: discord.Interaction):
...
+    # First day of the week (Monday) (https://stackoverflow.com/questions/39441639/getting-the-date-of-the-first-day-of-the-week/61743379#61743379)
+    monday = (dt.today() - timedelta(days=dt.today().weekday() % 7)).date()
+    friday = monday + timedelta(days=4)
+
+    # Create dictionary of events to sort
+    events = dict([(key, {}) for key in [x for x in range(5)]]) # cursed list comprehension
+    events_announced = 0
+
+    boilerplate = """@everyone
+# Howdy everyone! <:cowboysunglasses:1277767237037723751>
```
Comment (lilacstella) [inline cyberham/__main__.py:560]:
```suggestion
# Howdy everyone! <:sunglasses_cowboy:916376081576116354>
```

**Review by lilacstella — CHANGES_REQUESTED** [review]

**Comment by lilacstella** [comment]
honestly LGTM

**Review by lilacstella — APPROVED** [review]

**Review by HomeoStasis69 — APPROVED** [review]
looks good from my end

**Pull request merged**